src/libraries/System.ComponentModel.TypeConverter/src/System/ComponentModel/DateOnlyConverter.cs

```diff
@@ -9,30 +9,31 @@
 namespace System.ComponentModel
 {
     /// <summary>
-    /// Provides a type converter to convert <see cref='System.DateOnly'/>
-    /// objects to and from various other representations.
+    /// Provides a type converter to convert <see cref='System.DateOnly'/> objects to and from various other representations.
     /// </summary>
     public class DateOnlyConverter : TypeConverter
     {
         /// <summary>
-        /// Gets a value indicating whether this converter can convert an
-        /// object in the given source type to a <see cref='System.DateOnly'/>
+        /// Gets a value indicating whether this converter can convert an object in the given source type to a <see cref='System.DateOnly'/>
```

Comment:
The summary is now duplicated.

Also - were you going to switch around the other occurrences as well?

Reply:
I applied your suggestion which caused the duplication :-) I'll fix that. I'll keep the summary on the top as I saw that in the doc you sent.